turn/__init__.py

```diff
@@ -27,19 +27,38 @@ def do_request(self, data=None):
             data=json.dumps(data) if data is not None else None,
         )
 
+    def get_error(self, response):
+        if len(response.json()["errors"]):
+            return response.json()["errors"][0]
```

Comment:
nit, for a more pythonic version of this, I would have used a `try/except` block:
```python
try:
    return response.json()["errors"][0]
except (KeyError, IndexError):
    return None
```